FE/src/components/Account.jsx

```diff
@@ -177,19 +217,15 @@ export default function Account ({token}) {
                 //         "Authorization": `Bearer ${token}`,
                 //     },
                 // });
-                // const result = await response.json()             
+                // const result = await response.json();
+                // console.log(result);
                 if (token !== null) {
-                    setUserData(dummyUser);
-                    setRecipeList(dummyRecipes);
-                    setReviewList(dummyReviews);
-                    setCommentList(dummyComments);
+                    setUserData(dummyData);
                     setFail(false);
-                    setSuccess(true);
-                    // setUserBio(true);
-                    // setUserForm(false);
+                } if (dummyData.admin !== false){
```

Comment:
This condition might cause some problems once we start calling the API, because the response might come back but admin might be null. So I suggest changing to  if (dummyData.admin). But also I'm confused about the fact that dummyData is referenced at all outside the first condition on line 181. Once the real API call is implemented, I have a feeling this useEffect is gonna have to be reworked a little.